src/components/AccountPermissions/components/PermissionList/PermissionList.tsx

```diff
@@ -0,0 +1,75 @@
+import Skeleton from "@dashboard/components/Skeleton";
+import { PermissionData } from "@dashboard/permissionGroups/components/PermissionGroupDetailsPage";
+import {
+  Checkbox,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+} from "@material-ui/core";
+import { Box } from "@saleor/macaw-ui/next";
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { messages } from "../../messages";
+import { hasPermissionSelected } from "../../utils";
+
+interface PermissionListProps {
+  permissions: PermissionData[];
+  selectedPermissions: string[];
+  disabled?: boolean;
+  onPermissionChange: (key: string, value: boolean) => void;
+}
+
+export const PermissionList = ({
+  permissions,
+  onPermissionChange,
+  selectedPermissions,
+  disabled,
+}: PermissionListProps) => {
+  const intl = useIntl();
+
+  const render = () => {
```

Comment:
**Question**: I'm wondering if having this inside body `PermissionList` won't be more readable?

Instead of having `render()` you can have `Box` wrapper duplicated two times and I mine opinion is more readable than jumping back and forth.